### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by gracekabaghe**
- Books - the default view
- Added form for adding a book
- Categories "Under construction" text only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# the-book-store
\ No newline at end of file
+# Getting Started with Create React App
```
Comment (TedLivist) [inline README.md:1]: - [x] Kindly update your README file to be more descriptive of this project. I would recommend using this [Microverse template](https://github.com/microverseinc/readme-template)
- [x] Your project is missing the remaining redux files setup according to the image below. There should be a `books` folder with a `books.js` file. There should also be a `categories` folder with a `categories.js` file. You are also missing the `configureStore.js` file. Kindly add these files.
  
  grace-file

File: src/components/AddNewBook.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      </div>
+      <div className="bitem">
+        <h3>Dragons Den</h3>
+        <button type="button" className="removebtn">Remove</button>
+      </div>
+      <div className="bitem">
+        <h3>Bond</h3>
+        <button type="button" className="removebtn">Remove</button>
+      </div>
+    </div>
```
Comment (TedLivist) [inline src/components/AddNewBook.js:16]: - [x] There should be a **`Book`** component that takes in the **book data for a single book** as a prop and renders its title and author. Kindly extract a single book here into a new component and make it render props instead.
- [x] The new Book component should also have a remove **button**. This button does not have to work yet. It is merely a placeholder.

File: src/components/Home.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AddNewBook from './AddNewBook';
+
+function Home() {
```
Comment (TedLivist) [inline src/components/Home.js]: - [x] Let's use ES6 syntax for our function. Kindly change this to an arrow function

File: src/redux/Categories.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
```
Comment (TedLivist) [inline src/redux/Categories.js]: - [x] Change this to an arrow function

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+// function App() {
+//   return (
+//     <div className="App">
+//       <h1>The Bookstore</h1>
+//       <AddNewBook />
+//     </div>
+//   );
+// }
+
+// export default App;
```
Comment (TedLivist) [inline src/App.js]: - [x] Let's keep our code clean. Kindly remove commented code

File: src/components/Home.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+  return (
+    <div className="homePage">
+      <AddNewBook />
+    </div>
+  );
+}
+
+export default Home;
```
Comment (TedLivist) [inline src/components/Home.js]: - [x] There should be a component for rendering the list of books. I assume this is the component. Now, kindly import and render your created Book component here.

File: src/redux/Categories.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+  return (
+    <div>
+      <h1>Under Construction!</h1>
+    </div>
+  );
+}
+
+export default Categories;
```
Comment (TedLivist) [inline src/redux/Categories.js]: - [x] Our components should live in the components folder, not the redux folder. Kindly move this **`Categories`** component to the components directory.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gracekabaghe,

Good job so far!
✔️ Nice UI
✔️ Correct setup of npm packages
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Home.js (hunk 3 above)
Comment (gracekabaghe) [reply src/components/Home.js]: this is clearly your preference and not part of the requirements.

File: src/redux/Categories.js (hunk 4 above)
Comment (gracekabaghe) [reply src/redux/Categories.js]: Again your preferece

File: src/App.js (hunk 5 above)
Comment (gracekabaghe) [reply src/App.js]: the project is not even started yet, your preferences are petty

File: README.md (hunk 1 above)
Comment (gracekabaghe) [reply README.md:1]: done

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Grace, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**